instructure/canvas-android
# [MBL-16085][Student][Teacher] Initial theme selector bottom sheet

Test plan:
- Verify that the bottom sheet shows up at first start/update and theme selection works.
- Verify that it doesn't show up the second time, even if the user doesn't select anything and dismisses the bottom sheet.
- Check both apps.

refs: MBL-16085
affects: Student, Teacher
release note: none
